docs/source/getting_started/getting_started.md

```diff
@@ -111,7 +128,7 @@ curl -X PATCH "$URL/users/u1234/interactions" \
 ```
 
 ```{note}
-Please note that if an interaction between a user and a document is added, the document will **not** be part of the documents returned for future calls to the personalised endpoint.
+Please note that if an interaction between a user and a document is added, all snippets of a document will **not** be part of the documents returned for future calls to the personalised endpoint.
```

Comment:
:pushpin:  When updating this line I realized that semantic search is currently quite hard to use for anything which has documents with multiple snippets (except for a use case of refining existing search).

The problem is if you don't do semantic search based on previous search results you have no idea which snippet ids you want to use as the customer only knows the document ids and that there might be more then one snippet but not how many and which snippets there are :thinking: